xml/System.Buffers/StandardFormat.xml

```diff
@@ -384,9 +427,9 @@
         <Parameter Name="format" Type="System.ReadOnlySpan&lt;System.Char&gt;" />
       </Parameters>
       <Docs>
-        <param name="format">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="format">A read-only span that contains the character to parse.</param>
+        <summary>Converts a <see cref="T:System.ReadOnlySpan{System.Char}" /> into a <see cref="T:System.Buffers.StandardFormat" /> instance using <see cref="F:System.Buffers.StandardFormat.NoPrecision" /> precision.</summary>
```

Comment:
This cref doesn't exist. Should be T:System.ReadOnlySpan\`1 or maybe styled with the `<c>` tag if you want to render as System.ReadOnlySpan{System.Char}.